README.md

```diff
@@ -1,60 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio: setup and mobile version skeleton
 
-> One paragraph statement about the project.
+> For the first milestone in the process of creating your portfolio website, you will:
+- set up a new repository and prepare it for development using best practices (e.g. linters).
+- build the first 3 sections of the mobile website using the template you chose in the previous step
 
-![screenshot](./app_screenshot.png)
+![screenshot](.images./printScreen.png)
```

Comment:
Let's fix the screenshot.

Hint: 
I think you should double-check the name of the folder and how to set the path.
Seems you have it as `images` not `images.`

Let's try `![screenshot](./images/printScreen.png)` this should work just fine.

Reply:
It was my lack of attention. My mistake. Thanks to appoint me this ".". 

The other issue about, I'm still don't understanding what's happening. When I change for the first time works, If do I follow your step-by-step, doesn't work.

Reply:
@leonardoeng13 

Let me try my best to break it down, in my first comment up there, I didn't really talk about flex, I talked about Semantic HTML.

Now let me try my best to provide some hints via inline comments.